hathor/simulator/simulator.py

```diff
@@ -178,6 +178,7 @@ def create_artifacts(self, builder: Optional[Builder] = None) -> BuildArtifacts:
             .build()
 
         artifacts.manager.start()
+        self._clock.running = True
```

Comment:
Why do you have to manually mark it as running?

Reply:
For the same reason described in https://github.com/HathorNetwork/hathor-core/pull/694#discussion_r1286429491

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/694/commits/aa53857b5974f4082741ced56e763fb75de54e3d